src/content/docs/paper/dev/api/pdc.md

```diff
@@ -37,20 +38,35 @@ which is the object you want to store the data on. The third is the data itself.
 // Create a NamespacedKey
 NamespacedKey key = new NamespacedKey(pluginInstance, "example-key");
 
-ItemStack item = ItemStack.of(Material.DIAMOND);
-// ItemMeta implements PersistentDataHolder, so we can get the PDC from it
-item.editMeta(meta -> {
-    meta.getPersistentDataContainer().set(key, PersistentDataType.STRING, "I love Tacos!");
+World world = Bukkit.getServer().getWorlds().getFirst();
+
+PersistentDataContainer pdc = world.getPersistentDataContainer();
+
+pdc.set(key, PersistentDataType.STRING, "I love Tacos!");
+```
+
+[`ItemStack`](jd:paper:org.bukkit.inventory.ItemStack) however doesn't have this method and instead requires you to use its builder-style consumer:
+
+```java
+NamespacedKey key = ...; // Retrieve the key from before
+
+// For 1.20.4 and below, use 'new ItemStack(Material.DIAMOND)' instead
+ItemStack item = ItemType.DIAMOND.createItemStack();
+// ItemStack provides a util method, so we can directly edit its PDC
+item.editPersistentDataContainer(pdc -> {
+    pdc.set(key, PersistentDataType.STRING, "I love Tacos!");
 });
 ```
+Note: The [`ItemStack#editPersistentDataContainer()`](jd:org.bukkit.inventory.ItemStack#editPersistentDataContainer()) method on `ItemStack` is only available in 1.21.4+. For older versions, you need to access and modify the `ItemMeta` instead.
+For 1.16.5+, there's the [`ItemStack#editMeta()`](jd:org.bukkit.inventory.ItemStack#editMeta(Consumer)) method though.
```

Comment:
use an aside/admonition https://starlight.astro.build/guides/authoring-content/#note-aside

the Javadoc links are invalid, because they do not specify a project
```suggestion
:::note

The [`ItemStack#editPersistentDataContainer()`](jd:paper:org.bukkit.inventory.ItemStack#editPersistentDataContainer()) method on `ItemStack` is only available in 1.21.4+. For older versions, you need to access and modify the `ItemMeta` instead.
For 1.16.5+, there's the [`ItemStack#editMeta()`](jd:paper:org.bukkit.inventory.ItemStack#editMeta(Consumer)) method though.

:::
```

Reply:
that is ok